iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/IcebergAcidUtil.java

```diff
@@ -391,4 +393,17 @@ public T build() {
     }
   }
 
+  static long getTxnId() {
```

Comment:
that's not even required as it could be done as 1-liner
````
long txnId = Optional.ofNullable(SessionState.get())
    .map(ss -> ss.getTxnMgr().getCurrentTxnId()).orElse(0L);
````

Reply:
Reverted in #5716